Header header
uint16 thrusterChannel
bool thrusterOk
